src/client/containers/PullRequestSummary/ReviewSection/ReviewerSelection/index.js

```diff
@@ -3,178 +3,131 @@
 import React, { Component } from 'react'
 import Col from 'react-bootstrap/lib/Col'
 import Row from 'react-bootstrap/lib/Row'
-import fuzzy from 'fuzzy'
-import ListGroupItem from 'react-bootstrap/lib/ListGroupItem'
-import ListGroup from 'react-bootstrap/lib/ListGroup'
 import type { UserType } from 'universal/types'
-
-type ReviewerSelectionItemProps = {
-  isReviewer: boolean,
-  onToggleReviewer: (user: UserType) => void,
-  user: UserType,
-}
-
-const whiteSpaceRegExp = /\s+/g
-
-class ReviewerSelectionItem extends Component {
-
-  onToggleReviewer = () => {
-    this.props.onToggleReviewer(this.props.user)
-  }
-
-  props: ReviewerSelectionItemProps
-
-  render() {
-    const { isReviewer, user } = this.props
-
-    return (
-      <ListGroupItem
-        onClick={this.onToggleReviewer}
-        style={{
-          padding: '2px 10px',
-          textDecoration: 'none',
-          outline: 'none',
-          border: 'none',
-          boxShadow: 'none',
-        }}
-      >
-        <div
-          style={{
-            float: 'left',
-            marginRight: '12px',
-          }}
-        >
-          <i
-            className="fa fa-check"
-            style={{
-              visibility: isReviewer ? 'inherit' : 'hidden',
-              display: 'inline-block',
-              fontSize: '16px',
-              lineHeight: '40px',
-              width: '40px',
-              height: '40px',
-              textAlign: 'center',
-              verticalAlign: 'bottom',
-            }}
-          />
-        </div>
-        <div>
-          <div style={{ fontSize: '14px' }}>
-            <strong>{user.fullName}</strong>
-          </div>
-          <span style={{ color: 'grey', fontSize: '11px' }}>{user.username}</span>
-        </div>
-      </ListGroupItem>
-    )
-  }
+import Select from 'react-select'
+import MissingReviewerList from '../MissingReviewerList'
+import { connect } from 'react-redux'
+import { createSelector } from 'reselect'
+import { getPullRequest } from 'ducks/pullrequests/selectors'
+import './ReviewerSelection.css'
+
+type MissingReviewerType = {
+  area: string,
+  reviewers: Array<UserType>,
 }
 
-
 type ReviewersProps = {
+  id: string,
   reviewers: Set<string>,
   users: Array<UserType>,
   onToggleReviewer: (user: UserType) => void,
+  addReviewers: (id: string, users: Array<UserType>) => void,
+  missingReviewers: Array<MissingReviewerType>,
 }
 
+export const getMissingReviewers = createSelector(
+  getPullRequest,
+  (pr) => (pr && pr.missingReviewers ? pr.missingReviewers : [])
+)
+
+export type ValueType = {
+  label: string,
+  value: string,
+  user: UserType,
+}
 
 class ReviewerSelection extends Component {
   /* eslint-disable react/sort-comp */
 
   constructor(props: ReviewersProps) {
     super(props)
     this.state = {
-      usersAfterSearch: this.props.users,
-      query: '',
+      value: [],
```

Comment:
Slightly confusing, I agree. Was to keep consistent with the naming of the select control.